internal/pkg/config/config.go

```diff
@@ -48,6 +48,7 @@ type Params struct {
 	SiderolinkWireguardBindAddress       string `yaml:"siderolinkWireguardBindAddress"`
 	SiderolinkWireguardAdvertisedAddress string `yaml:"siderolinkWireguardAdvertisedAddress"`
 	SiderolinkDisableLastEndpoint        bool   `yaml:"siderolinkDisableLastEndpoint"`
+	SiderolinkUseGRPCTunnel              bool   `yaml:"siderolinkUseGRPCTunnel`
```

Comment:
@Unix4ever your forced pushed removed this mistake i fixed before. Can i just push another commit or do you want to change that yourself? Just trying to make sure the commit doesnt get erased again haha

```suggestion
	SiderolinkUseGRPCTunnel              bool   `yaml:"siderolinkUseGRPCTunnel"`
```

Reply:
yeah, good catch. Fixed